### PR TITLE
Details worker

### PR DESCRIPTION
**Pull request by AsalaKM**
### Making the worker details page.

- Add new routs #104  #155
>  _/worker/:id
- Add new query details .. #103
-  Relates #28   #132

### CODE REVIEW

File: src/client/src/components/pages/workers/workerDetails/index.jsx [hunk 1]
```diff
@@ -19,6 +20,19 @@ export default class index extends Component {
     this.props.history.push('/workers/view')
   };
 
+  getDetails = async () => {
+    const id = this.props.match.params.id;
+    const data = await axios(`/api/v2/worker/${id}`);
+    const details = data.data.details;
+    console.log(details)
```
Comment (ishak52) [inline src/client/src/components/pages/workers/workerDetails/index.jsx]: Delete `console.log()`

File: src/client/src/components/pages/workers/workerDetails/index.jsx [hunk 2]
```diff
@@ -19,6 +20,19 @@ export default class index extends Component {
...
+    const date = details.date_of_birth.split("T")[0];
+    console.log(data);
```
Comment (ishak52) [inline src/client/src/components/pages/workers/workerDetails/index.jsx]: Delete `console.log()`

File: src/client/src/components/pages/workers/workerDetails/index.jsx (hunk 1 above)
Comment (AsalaKM) [reply src/client/src/components/pages/workers/workerDetails/index.jsx]: done :+1:

**Comment by tomduggan85** [comment]
Also looks good!

**Pull request merged**